src/main/webapp/app/course/learning-paths/pages/learning-path-student-page/learning-path-student-page.component.ts

```diff
@@ -0,0 +1,61 @@
+import { Component, Signal, computed, inject, viewChild } from '@angular/core';
+import { LearningPathService } from 'app/course/learning-paths/learning-path.service';
+import { LearningObjectType } from 'app/entities/competency/learning-path.model';
+import { Observable, catchError, map, of, switchMap } from 'rxjs';
+import { onError } from 'app/shared/util/global.utils';
+import { HttpErrorResponse } from '@angular/common/http';
+import { toObservable, toSignal } from '@angular/core/rxjs-interop';
+import { CommonModule } from '@angular/common';
+import { LearningPathStudentNavComponent, LoadedValue } from 'app/course/learning-paths/components/learning-path-student-nav/learning-path-student-nav.component';
+import { AlertService } from 'app/core/util/alert.service';
+import { ActivatedRoute, RouterModule } from '@angular/router';
+import { CourseExerciseDetailsModule } from 'app/overview/exercise-details/course-exercise-details.module';
+import { LearningPathLectureUnitComponent } from 'app/course/learning-paths/components/learning-path-lecture-unit/learning-path-lecture-unit.component';
+import { LearningPathExerciseComponent } from 'app/course/learning-paths/components/learning-path-exercise/learning-path-exercise.component';
+
+@Component({
+    selector: 'jhi-learning-path-student-page',
+    templateUrl: './learning-path-student-page.component.html',
+    styleUrl: './learning-path-student-page.component.scss',
+    standalone: true,
+    imports: [CommonModule, RouterModule, LearningPathStudentNavComponent, CourseExerciseDetailsModule, LearningPathLectureUnitComponent, LearningPathExerciseComponent],
```

Comment:
Mixing standalone components and modules is probably going to result in a few `imports` declarations like these. I wonder if we should hold off on that for now and decide this globally

Reply:
Probably its best to use standalone components for new components and migrate older ones one by one to standalone (e.g. Let's see what the maintainers thoughts on this is)

Reply:
already talked to Stephan, he is in favour of using standalone components. I will take this to the next DEV meeting.